sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -92,34 +92,78 @@ class BeamFnControlServicer(beam_fn_api_pb2_grpc.BeamFnControlServicer):
   _DONE_MARKER = object()
 
   def __init__(self):
-    self._push_queue = queue.Queue()
     self._futures_by_id = dict()
-    self._read_thread = threading.Thread(
-        name='beam_control_read', target=self._read)
+    self._read_thread = None
     self._uid_counter = 0
     self._state = self.UNSTARTED_STATE
     self._lock = threading.Lock()
+    self._multi_worker_worker_handler = None
```

Comment:
I think we don't need to have this here anymore with the other proposed changes.